docs: note ES|QL views in Kibana editor autocomplete (#5907) (#6085)
elastic/docs-content: explore-analyze/query-filter/languages/esql-kibana.md

Every {{esql}} query starts with a [source command](elasticsearch://reference/query-languages/esql/esql-commands.md#esql-source-commands) that retrieves data:

> - [`FROM`](elasticsearch://reference/query-languages/esql/commands/source-commands.md#esql-from) retrieves data from data streams, indices, or aliases.

Author: ```suggestion
- [`FROM`](elasticsearch://reference/query-languages/esql/commands/source-commands.md#esql-from) allows you to define the data sources to query by specifying data streams, [{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md), indices, or aliases.
```

>   - {applies_to}`stack: preview 9.4.0` {applies_to}`serverless: unavailable` `FROM` also accepts [{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md) as sources.

Author: ```suggestion
```

![The ES|QL syntax reference and the autocomplete menu](/explore-analyze/images/kibana-esql-in-app-help.png "")

> {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` In **Discover**, the editor includes interactive browsers for selecting data sources and field names from the autocomplete menu. Refer to [](/explore-analyze/discover/try-esql.md#discover-esql-resource-browsers) for details.

Author: ```suggestion
{applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` In **Discover**, the editor includes interactive browsers for selecting data sources and field names from the autocomplete menu. Refer to [](/explore-analyze/discover/try-esql.md#discover-esql-resource-browsers) for details. [{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md) aren't included in the data source browser. They are available only from the autocomplete menu.
```

> {applies_to}`stack: preview 9.4.0` [{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md) aren't shown in the data source browser. The autocomplete suggests them as sources in `FROM` commands, alongside indices, data streams, and aliases.

Author: ```suggestion
```
